setup.py

```diff
@@ -44,6 +44,8 @@ def main():
                         help="Install dependencies for PCL extension and exit")
     parser.add_argument("--clean-build", action='store_true',
                         help="Remove build directory before cmake")
+    parser.add_argument("--with-all", action='store_true',
+                        help="Build RGL with all possible extensions, tools and tests.")
```

Comment:
At the moment, only extensions are handled with this flag. Add flags to enable building tools and tests. Or modify the help message.